sdk/iot/hub/samples/README.md

```diff
@@ -46,16 +49,27 @@ For more details on X509 authentication, refer to [this online documentation](ht
 
 ## Samples
 
-The following section documents various samples. All of them use the [Eclipse Paho MQTT C client][Eclipse_Paho].
-For all of these samples, the following environment variables will need to be set:
+For all of these samples, the following environment variables will need to be set.
 
+**NOTE**: These need to be set before running the cmake commands.
+
+- `VCPKG_DEFAULT_TRIPLET`: The triplet created using the instructions [here][VCPKG_DIRECTIONS].
```

Comment:
Duplicating these instructions will likely lead to issues in the future when contributing.md changes (I don't think we'll maintain both places).
We should just point to VCPKG_Directions saying those need to be done initially.

Reply:
I'm not duplicating the instructions here, only stating that the env var needs to be set. I understand your point but I want to be explicit everything that needs to be set in order for these to work. If I don't, someone will try to compile, paho linking will fail, and they'll have to hunt around everywhere for how to resolve that. I tried to be as minimal as possible about the details and reroute people elsewhere